style.css

```diff
@@ -102,3 +113,312 @@ button:active {
   background: #2230d2;
   color: #fff;
 }
+
+/* Works */
+.works {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 88px;
+  position: absolute;
+  width: 375px;
+  height: 2788px;
+  top: 718px;
+  background: #dfe1e6;
+}
+
+.card-works1 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  height: 552px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.card-works2 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  height: 596px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.card-works3 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  height: 552px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.card-works4 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  gap: 12px;
+  width: 327px;
+  height: 596px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
```

Comment:
- [ ] In order to follow the DRY method of the best practice in coding, (i.e Don't Repeat Yourself). t is better to have a shorter line of code and be sure you aren't repeating yourself. I notice while trying to style your work-cards, you have written the same line of codes for all four cards in a separate CSS class. The only property that seems to be different here is the `height`, I would kindly implore you to give all the four cards the same class name and use that single class name for styling all the cards.

Example :
```CSS
.card-works {
  display: flex;
  flex-direction: column;
  align-items: center;
  padding: 16px;
  gap: 12px;
  width: 90%;
  margin: 0 auto;
  height: auto;
  background: #fff;
  border: 1px solid #dfe1e6;
  border-radius: 16px;
}
```

Instead of having it as `.card-works1`, `.card-works2`, `.card-works3`, and `.card-works4`